lib/big_endian_span_test.cc

```diff
@@ -1,3 +1,18 @@
+/*
```

Comment:
Let's use the short version of the ISC header.

Also add the SPDX identifier.

Reply:
@mithro How does a short version look like?

Reply:
See first version in https://github.com/SymbiFlow/prjxray/issues/1300

Reply:
Done